### ISSUE TITLE
Running tweakreg with updatehdr=True archives an extra copy of the tweaked WCS

### ISSUE THREAD

**Issue by mcara** [spacetelescope/drizzlepac#690]
Running `tweakreg` on an image archives the primary WCS as alternative WCS `'A'`, updates primary WCS with the new solution, **AND** creates a second alternative WCS `'B'` with the same `WCSNAME` as the primary WCS (`'TWEAK'`).

I think tweakreg should not archive the **new** primary WCS - only the the "old" one (as `'A'`).

CC: @stsci-hack

**Comment by mcara** [comment]
The way I run the code is:

```python
tweakreg.TweakReg('ie6d07ujq_flc.fits', refcat='refcat.txt', refxyunits='pixels', 
fitgeometry='shift', imagefindcfg={'threshold': 1000}, use2dhist=False, 
minobj=1, searchrad=100, searchunits='pixels', interactive=False,
residplot='No', updatehdr=True, wcsname='TWEAK1')
```

Reference catalog was a copy of the image catalog: `ie6d07ujq_flc_refxy_catalog.coo --> refcat.txt`. Here I am not concerned with matching and accuracy, just the machinery of updating and archiving WCSes.

**Comment by stsci-hack** [comment]
Resolved with #697 on 26-Jun-2020
